uber/models.py

```diff
@@ -439,14 +439,18 @@ def order(self, attrs):
                 order.append(col.desc() if attr.startswith('-') else col)
             return self.order_by(*order)
 
-        def icontains(self, attr=None, val=None, **filters):
-            query = self
+        def icontains_predicate(self, attr=None, val=None, **filters):
```

Comment:
A docstring would be awesome. I have never encountered this use of the word 'predicate,' for example.